Makefile

```diff
@@ -28,11 +30,15 @@ ui/lib/src/api: swagger.yaml
 
 .PHONY: lint
 lint:
-	golint $(shell go list ./pkg/gitpods/...)
+	$(GO) mod tidy
+	$(GO) mod vendor
+	git update-index --refresh
+	git diff-index --quiet HEAD
```

Comment:
`update-index --refresh` so that touched-but-unchanged don't mess up `diff-index` :roll_eyes:

Reply:
Also, those are plumbing which means they have proper exit-codes that `make` understands... (hence not `diff --quiet`)

Reply:
As these new commands do a lot more, can we put them in a different make target? In CI I'm fine with running both make targets in the same lint step though.